beagle_etl/celery.py

```diff
@@ -17,7 +17,7 @@
 # Load task modules from all registered Django app configs.
 app.autodiscover_tasks()
 
-app.conf.task_always_eager = settings.DEBUG
+# app.conf.task_always_eager = settings.DEBUG
```

Comment:
was this the setting that allowed for Celery tasks to run synchronously during debugging? That was really helpful for dev and debug